spec/presenters/idv/in_person/ready_to_verify_presenter_spec.rb

```diff
@@ -120,6 +122,35 @@
     end
   end
 
+  describe '#partner_agency' do
+    subject(:partner_agency) { presenter.partner_agency }
+
+    it 'returns partner agency' do
+      expect(partner_agency).to eq('http://localhost:3000')
+    end
+  end
+
+  describe '#days_remaining' do
+    subject(:days_remaining) { presenter.days_remaining }
+    let(:config) { IdentityConfig.store.in_person_enrollment_validity_in_days }
+
+    context '4 days until due date' do
+      it 'returns 3 days' do
+        travel_to(enrollment_established_at + (config - 4).days) do
+          expect(days_remaining).to eq(3)
+        end
+      end
+    end
+
+    context '11 days until due date' do
+      it 'returns 10 days' do
+        travel_to(enrollment_established_at + (config - 11).days) do
+          expect(days_remaining).to eq(10)
+        end
+      end
+    end
```

Comment:
Since this is so similar to the above test for 4 days it seems unnecessary to me